src/sentry/integrations/utils/code_mapping.py

```diff
@@ -234,7 +256,14 @@ def _generate_code_mapping_from_tree(
 
         stack_path = frame_filename.raw_path
         source_path = matched_files[0]
-        stack_root, source_root = find_roots(stack_path, source_path)
+
+        try:
+            stack_root, source_root = find_roots(stack_path, source_path)
+        except Exception:
```

Comment:
can we find a more specific `Exception` class to except on here as well?

Reply:
Fixed in the most recent commit, thanks for the keen eye!